apps/dashboard/test/system/remote_files_test.rb

```diff
@@ -82,13 +82,11 @@ class RemoteFilesTest < ApplicationSystemTestCase
         find('#clipboard-copy-to-dir').click
 
         # files are copying but it takes a little while
-        find('tbody a', exact_text: 'app', wait: MAX_WAIT)
         find('tbody a', exact_text: 'config', wait: MAX_WAIT)
         find('tbody a', exact_text: 'manifest.yml', wait: MAX_WAIT)
 
         # with copying done, let's assert on the UI and the file system
         assert_selector 'span', text: '100% copy files', count: 1
-        assert_equal '', `diff -rq #{File.join(dir, 'app')} #{Rails.root.join('app')}`.strip, 'failed to recursively copy app dir'
```

Comment:
`rclone` won't copy symlinks. This PR adds a symlink in the `app/views` directory, so we can't really test it anymore.